README.md

```diff
@@ -1,6 +1,7 @@
 # NTP Mock Server
 
-[![Build Status](https://travis-ci.org/dmcnulla/ntp.svg?branch=master)](https://travis-ci.org/dmcnulla/ntp)
+[![Build Status](https://travis-ci.org/dmcnulla/ntp.svg?branch=coverage)](https://travis-ci.org/dmcnulla/ntp)
```

Comment:
That just says to provide the report for this branch in this readme. In the master, it will be master.